model/src/main/java/org/projectnessie/model/CommitMeta.java

```diff
@@ -31,15 +34,46 @@
   @Nullable
   public abstract String getHash();
 
-  public abstract String getCommiter();
+  @Value.Default
+  public String getCommiter() {
```

Comment:
defaults are used to help clients quickly construct commit meta. Can be reset for consistency in REST client though. Committer is the system of record/auditable committer as it is set by REST server w/o overrides.